src/css/components/navigation.less

```diff
@@ -26,6 +26,7 @@
 .navbar-inverse .navbar-nav > .active > a:focus {
   color: @black-haze;
   background-color: transparent;
+  cursor: pointer;
```

Comment:
Some follow up context.

The reason this exists is because we're using Bootstrap's `nav-tabs` class on our nav bar.

No pointer cursor makes more sense if these were tabs. But in our use case, it's a top level nav bar, and should behave like a nav bar.

Reference
* http://getbootstrap.com/components/#nav-tabs
* https://github.com/angular-ui/bootstrap/issues/475